package.json

```diff
@@ -8,17 +8,17 @@
     "node": ">=12.17"
   },
   "scripts": {
-    "start": "node index.js",
-    "start:prod": "NODE_ENV=production node index.js",
-    "start:scheduler": "node schedule.js",
-    "start:scheduler:prod": "NODE_ENV=production node schedule.js",
-    "setup": "node scripts/setup.js",
+    "start": "node --experimental-json-modules index.js",
```

Comment:
I would prefer that we use [`createRequire`](http://nodejs.org/api/modules.html#modules_module_createrequire_filename) when needed, [as recommended](https://medium.com/@nodejs/announcing-a-new-experimental-modules-1be8d2d6c2ff#30).

Reply:
I do not see where it's recommended. 
They recommend to use createRequire instead of `--experimental-modules` but here I use `--experimental-json-modules` and about this one, they said:

> A separate flag --experimental-json-modules enables experimental support for import of JSON files. There is work in progress for standardizing this feature with browsers, and Node.js hopes to align our support with the eventual standard.

Do you still prefer recreating the require function?

Reply:
Moreover, it's only required for the `npm run validate` command.

Reply:
> I do not see where it's recommended.

You're right, I'm sorry, the second paragraph you mentioned does add details to the initial statement.

> Do you still prefer recreating the require function?

Yes, considering the stage of advancement of the standardisation for JSON modules (see https://github.com/whatwg/html/issues/4315), I would feel much safer relying on a stable API rather than a flag whose behavior is less specified.

Reply:
Okay :)